demo_scripts/Classes/custom_menu_bar_class.py

```diff
@@ -79,7 +79,10 @@ def _delete_top_button(self):
         list_of_all_menus_as_dicts = qtm.gui.get_menu_items()
         for curr_menu in list_of_all_menus_as_dicts:
             if curr_menu["submenu"] == self._menu_id:
-                print("\n" + "The '" + qtm.gui.get_menu_item(curr_menu["submenu"], 0)["text"] + "' button was successfully removed.")
+                if qtm.gui.get_menu_item(curr_menu["submenu"], 0)["text"] == "" and qtm.gui.get_menu_item(curr_menu["submenu"], 0)["command"] == "":
```

Comment:
question: is there any other way to get this than a separator?

Reply:
I don't believe it is possible, no, since trying to create a command with an empty string for a name results in the following error message:
**RuntimeError - error in method 'qtm.gui.add_command': command cannot be an empty string**